x-pack/legacy/plugins/maps/public/embeddable/types.ts

```diff
@@ -0,0 +1,34 @@
+/*
```

Comment:
Can you move all of these type definitions to https://github.com/elastic/kibana/blob/master/x-pack/legacy/plugins/maps/public/layers/tooltips/tooltip_property.ts so they live closer to where they are used. Then map_embeddable/index can just export them from that folder.